lib/replies.js

```diff
@@ -42,24 +42,20 @@ module.exports.invalidWhyParticipated = function invalidWhyParticipated(req, res
   return sendResponse('invalidWhyParticipated', req, res);
 };
 
-module.exports.menuCompleted = function menuCompleted(req, res) {
-  return sendResponse('completedMenu', req, res);
+module.exports.completedPhotoPost = function completedPhotoPost(req, res) {
+  return sendResponse('completedPhotoPost', req, res);
 };
 
-module.exports.invalidCmdCompleted = function invalidCmdCompleted(req, res) {
-  return sendResponse('invalidCompletedMenuCommand', req, res);
+module.exports.completedPhotoPostAutoReply = function completedPhotoPostAutoReply(req, res) {
+  return sendResponse('completedPhotoPostAutoReply', req, res);
 };
 
-module.exports.menuSignedUp = function menuSignedUp(req, res) {
-  return sendResponse('gambitSignupMenu', req, res);
+module.exports.startPhotoPost = function startPhotoPost(req, res) {
+  return sendResponse('startPhotoPost', req, res);
 };
 
-module.exports.externalSignupMenu = function externalSignupMenu(req, res) {
```

Comment:
This is a pre-TGM relic, a `POST /campaignActivity` request would never return an `externalSignupMenu` reply template anymore, confirmation messages for web signups are now handled via Gambit Conversations `POST /v2/messages?origin=signup`.